pkg/api/v20170701/types.go

```diff
@@ -215,29 +215,31 @@ func (a *AgentPoolProfile) UnmarshalJSON(b []byte) error {
 	return nil
 }
 
-// OrchestratorType defines orchestrators supported by ACS
```

Comment:
@JiangtianLi 

I was thinking to remove OrchestratorType custom type in my previous `validate tag` PR. Now i take it into action. And also enforce the same tradition across the old versions.